remove typo from step 2 to create a load balancer

laraaleite opened the pull request:
BEFORE:   - **Public** - for the type of the load balancer.'s fonrt 
AFTER:      - **Public** - for the type of the load balancer.

azurecla commented:
Hi __@v-cmans__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-cmans). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

laraaleite committed:
remove typo from step 2 to create a load balancer

BEFORE:   - **Public** - for the type of the load balancer.'s fonrt 
AFTER:      - **Public** - for the type of the load balancer.
